package.json

```diff
@@ -152,19 +157,8 @@
     "menus": {
       "commandPalette": [
         {
-          "command": "ocaml.select-sandbox"
-        },
-        {
-          "command": "ocaml.server.restart"
-        },
-        {
-          "command": "ocaml.open-terminal"
-        },
-        {
-          "command": "ocaml.open-terminal-select"
-        },
-        {
-          "command": "ocaml.switch-impl-intf"
+          "command": "dune.current-dune-file",
+          "when": "editorLangId  == ocaml || editorLangId  == ocaml.interface || editorLangId == reason || editorLangId == ocaml.ocamllex || editorLangId == ocaml.menhir"
```

Comment:
Why is this command disabled for non source files? It should work in any context.

Reply:
The command says 'open "current" dune file', where "current" is understood as the dune component (lib/exec) that the source file is a part of. Q: What would you expect "current dune file" be for, say, a `package.json` in the project root? I was trying to be strict here to not confuse users.
---
All in all, I've been using this command for some time and find it super useful! It could, however, be improved by jumping directly to the dune stanza/component, which the source is a component of.

Reply:
>  Q: What would you expect "current dune file" be for, say, a package.json in the project root? I was trying to be strict here to not confuse users.

If there's a dune file in the same dir as `package.json`, I would expect it to jump there. Note that one could have a dune file for C, C++, Coq, cram, and all kinds of other source files. So if anything, sticking to a subset is equally confusing.

> It could, however, be improved by jumping directly to the dune stanza/component, which the source is a component of.

Yes, that would be nicer. Especially given that the source file might not belong to any dune file and the command would still jump to a dune file.

Reply:
Will update to account for any open file then.